vcpkg/overlay/qgis-qt6/wrongattributeerrormessage.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/core/providers/ogr/qgsogrprovider.cpp b/src/core/providers/ogr/qgsogrprovider.cpp
+index 8b0ac5a4cc9..c8890ebd36d 100644
+--- a/src/core/providers/ogr/qgsogrprovider.cpp
++++ b/src/core/providers/ogr/qgsogrprovider.cpp
+@@ -1738,7 +1738,7 @@ bool QgsOgrProvider::addFeaturePrivate( QgsFeature &f, Flags flags, QgsFeatureId
+ 
+       if ( !ok )
+       {
+-        pushError( tr( "wrong data type for attribute %1 of feature %2: %3" ).arg( qgisAttributeId ) .arg( f.id() ).arg( qType ) );
++        pushError( tr( "wrong data type for attribute %1 (%2) of feature %3" ).arg( mAttributeFields.at( qgisAttributeId ).name(), OGR_GetFieldTypeName( type ), QString::number( f.id() ) ) );
```

Comment:
```suggestion
+        pushError( tr( "Could not convert value for attribute \"%1\" to type %2" ).arg( mAttributeFields.at( qgisAttributeId ).name(), OGR_GetFieldTypeName( type ) ) );
```

Let's go for this, the feature id does not help at all and is already added by a different place in the error stack.